src/main.js

```diff
@@ -1,5 +1,8 @@
 // Create variables targetting the relevant DOM elements here 👇
 
+var homeButton = document.querySelector('.home-button');
+var randomCoverButton = document.querySelector('.random-cover-button');
+var saveCoverButton = document.querySelector('.save-cover-button');
```

Comment:
Looking great! Thanks for taking the lead on figuring this portion out!

Reply:
Thanks Daniel! :)